app/models/concerns/purchase/blockable.rb

```diff
@@ -103,7 +103,7 @@ def recent_stripe_fingerprint
     end
 
     def blockable_emails_if_fraudulent_transaction
-      [purchaser&.email, email, gifter_email].compact_blank.uniq
+      [purchaser&.email, purchaser&.paypal_payout_email, email, gifter_email].compact_blank.uniq
```

Comment:
@EmCousin _a I'm a little confused about the requirement 🤔 We need to block based on buyer's PayPal email address, right? `User#paypal_payout_email` is the email address we use to payout to the seller. Could you please clarify with @michellelarney?